unittests/scans/csaf/csaf_one_vuln.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Nit: [The keys in the CSAF document should be sorted alphabetically.](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html#53-sorting). (Applies to all.) If you use the "CSAF Document" tab in Secvisogram the keys are sorted automatically. Then, hit "Export CSAF".